File: src/test/scala/fr/ramiro/gocd/dns/poller/GoPluginDnsPollerTest.scala
package fr.ramiro.gocd.dns.poller

import org.scalatest.FunSuite

class GoPluginDnsPollerTest extends FunSuite {
  test("validateRepositoryConfiguration"){
    val result = new GoPluginDnsPoller().validateRepositoryConfiguration(Map("RAMDOM" -> "rand"), GoPluginDnsPoller.repositoryFields)
    println(result)
  }
}

File: src/test/java/fr/ramiro/gocd/dns/poller/GoPluginDnsPollerTest.java
package fr.ramiro.gocd.dns.poller;

import org.junit.Test;
import static org.junit.Assert.*;

public class GoPluginDnsPollerTest {

    @Test
    public void testInit(){
        GoPluginDnsPoller plugin = new GoPluginDnsPoller();
        assertNotNull(plugin);
    }
}
